modules/lambda_teams_alert_monitor_log/README.md

```diff
@@ -36,3 +36,4 @@ module "monitor_log" {
 }
```

Comment:
Why have `environmentname` as a variable here? Is it to be able to reuse the module with unique names in the same account? I think in stead of having two variables, you could just have one. Even better, you could make it optional and just generate some random string to make unique names by default.

Reply:
The reason to have it is to be able to spin up dev and prod infrastructure in separate states but from the same configuration. From [here](https://learn.hashicorp.com/tutorials/terraform/organize-configuration) it appears to be best practice to set the environmentname in dev.tfvars and prod.tfvars files. I guess I will do that

Reply:
I haven't read the article you linked, but I'd say `*.tfvars` is useful for clients using the module, not the module itself. The client can also use terragrunt to achieve the same if they want.

Reply:
That's what I meant. The .tfvars will be used by the client. You are right, the module should not care. I tend to confuse the module with the client because I am testing as a client